www/vorj00/project_01/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="description" content="Berkshire Hathaway official website">
+    <meta name="keywords" content="Berkshire Hathaway,investing,website,webpage,official">
+    <meta name="author" content="Jakub Voracek">
+    <title>Home | Berkshire Hathaway</title>
+    <link rel="shortcut icon" href="assets/img/favicon.ico">
+    <link rel="icon" type="image/png" sizes="192x192" href="assets/img/favicon.png">
+    <link rel="icon" type="image/x-icon" href="assets/img/favicon.svg">
+    <link rel="stylesheet" href="assets/css/components.index.css">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
```

Comment:
kdyz jsem byl poprve na hlavni strance tak jsem se zasekl, ... neni to moc navodny, a kdyz rozkliknu odkazy tak bud se dostanu na jinej web nebo na stranku, kde se dozvim skoro nic, proste mi prijde  ze je tam malo informaci... je to tady takove mrtvo

hlavni stranka ma slouzit jako rozcestnik, tam se nachazeji dulezite a dojemne informace, pripadne zrejme odkazy na ne. normalni uzivatel by nevedel, o cem je ten web. neni to nic osobniho... ale prvni dojem je dulezity.

doporucuji tedy tam pridat par widgetu o tom, co konglomerat dela, jake ma uspechy, jak je bohaty apod, nebo aspon zrejme odkazy kam by se uzivatel ma dostat dal. jinak by byla skoda tech kotel css stylu co jsi dal dohromady :)